src/deferred/exceptionHook.js

```diff
@@ -7,14 +7,20 @@ define( [
 
 // These usually indicate a programmer mistake during development,
 // warn about them ASAP rather than swallowing them by default.
-var rerrorNames = /^(Eval|Internal|Range|Reference|Syntax|Type|URI)Error$/;
+// Notice that Error is a choice too, as ? regex means (0-1) times.
+var rerrorNames = /^(Eval|Internal|Range|Reference|Syntax|Type|URI)?Error$/;
 
 jQuery.Deferred.exceptionHook = function( error, stack ) {
 
 	// Support: IE 8 - 9 only
 	// Console exists when dev tools are open, which can happen at any time
 	if ( window.console && window.console.warn && error && rerrorNames.test( error.name ) ) {
-		window.console.warn( "jQuery.Deferred exception: " + error.message, error.stack, stack );
+		window.console.warn( "jQuery.Deferred exception: " + error.message, error.stack );
+
+		// If process stack exists, then give it out as a second warning too.
+		if ( stack ) {
+			window.console.warn( stack );
+		}
```

Comment:
I don't think it's a problem that it logs `undefined`. A real problem here is that source maps are not applied to the error stack, see #3179 & https://bugs.chromium.org/p/chromium/issues/detail?id=622227#c3. We should try various solutions, including the one proposed in the Chromium bug but it's better to do it in a separate PR if you'd like to try to fix that.